util/admin/admin_test.go

```diff
@@ -226,7 +214,7 @@ func (s *testSuite) TestCancelJobs(c *C) {
 	errs, err = CancelJobs(txn, []int64{-1})
 	c.Assert(err, IsNil)
 	c.Assert(errs[0], NotNil)
-	c.Assert(errs[0].Error(), Equals, "[admin:4]DDL Job:-1 not found")
+	c.Assert(errs[0].Error(), Matches, "*DDL Job:-1 not found")
```

Comment:
```suggestion
	c.Assert(ErrDDLJobNotFound.Equal(errs[0])), IsTrue)
```

Reply:
`c.Assert(errs[0].Error(), Matches, "*DDL Job:-1 not found") ` also check the job ID， I think we should keep it.